### PR TITLE
 [WIP] Convert json id from https://signalk.github.io/specification/schemas…

### PR DESCRIPTION
**Pull request by rob42**
Convert json id from https://signalk.github.io/specification/schemas/ to https://signalk.org/specification/schemas/v1/

This includes all instances (including docs) in master, but probably doesnt address misc pulls, and maybe gh-pages.  All tests still pass :-)

### CODE REVIEW

**Comment by tkurki** [comment]
Is there something left to do, as the PR name is [WIP]?

**Comment by rob42** [comment]
Was waiting for #357

**Comment by tkurki** [comment]
I don't quite get 0330e14b449d61056261dfa53f5deac54eb4cec6 : you add a schema:version task that replaces npm_package_version with the correct version, but at the same time modify all the schemas so that they have hardcoded `0.9.0-SNAPSHOT`in them.

File: index.js [hunk 1]
```diff
@@ -39,23 +39,23 @@ var FullSignalK = require('./src/fullsignalk');
 function getTv4() {
   var tv4 = require('tv4');
   var vesselSchema = require('./schemas/vessel.json');
-  tv4.addSchema('https://signalk.github.io/specification/schemas/vessel.json', vesselSchema);
+  tv4.addSchema('https://signalk.org/specification/schemas/0.9.0-SNAPSHOT/vessel.json', vesselSchema);
```
Comment (tkurki) [inline index.js]: Let's extract a constant here and not duplicate (my copypaste, I know...)

**Comment by tkurki** [comment]
I take back my comments, as I finally understand that `schema:version` will replace schema versions with the one from the package.json, whatever they are. The constant extraction is not really relevant either.

**Pull request merged**